docs/testing/01-overview.md

```diff
@@ -30,6 +30,7 @@ expects to receive the event `Z`.
 Stencil currently supports the following tools for testing components:
 
 - [__Stencil Test Runner__](./stencil-testrunner/01-overview.md): a built-in test runner based on Jest for unit and end-to-end testing with Puppeteer to run within an actual browser in order to provide more realistic results.
+- [__WebdriverIO__](./webdriverio/01-overview.md): the next-gen browser and mobile automation test framework for Node.js that allows you to run component and end-to-end tests across all browser.
```

Comment:
This feels a little bit like marketing copy for webdriverio, maybe we can reword to highlight very briefly some of what this would offer that you don't get with e.g. puppeteer?

```suggestion
- [__WebdriverIO__](./webdriverio/01-overview.md): the next-gen browser and mobile automation test framework for Node.js that allows you to run component and end-to-end tests across all browsers.
```

Reply:
I copied from the main website but yeah, we should stay away from marketing/advertising phrases